Proxy/proxy.py

```diff
@@ -448,6 +565,7 @@ def sqlInsert(data):
 
 def main():
     log = proxyConfig.getboolean('PROXY', 'LOG')
+    testmode = proxyConfig.getboolean('PROXY', 'TESTMODE')
```

Comment:
Just noting that we should probably fail gracefully on this with a try/except on config read but I won't hold up the PR for this as I am pretty sure `Proxy` hasn't received attention to this at all anyways.